jmxtrans-utils/src/main/java/com/googlecode/jmxtrans/util/NumberUtils.java

```diff
@@ -43,4 +43,26 @@ public static boolean isNumeric(Object value) {
 		return false;
 	}
 
+	public static boolean isValidNumber(Object value) {
+		if (!(value instanceof Number)) {
```

Comment:
True.  I had really focused this on instances of Numbers that would be returned from JMX, not a string that would be converted.

I can address this.